.drone.star

```diff
@@ -150,7 +153,7 @@ def main(ctx):
 
     pipelines = before + [ notify_pipeline ]
 
-  elif '[docs-only]' in (ctx.build.title + ctx.build.message):
+  elif '[docs-only]' in ctx.build.title:
```

Comment:
PR messages mentioning [docs-only] should also be capable of running a full CI run. This eg. broke this PR completely, so I changed it.